docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Use Ibexa Engage to collect customer data by creating interactive content.
+---
+
+# [[= product_name_engage =]]
+
+[[= product_name_engage =]] is a data collection tool.
+It gives you the ability to use the Qualifio tools to engage your audiences. You can use interactive content to build relationships and collect important data, for example, a list of recent orders, or personal information about customers.
```

Comment:
This might be a first time people hear about Qualifio - I believe linking to their website (as we do in the "install Enage" page) would be helpful for them